src/utils/Parsing.fsi

```diff
@@ -84,20 +100,29 @@ type internal Tables<'tok> =
 
     /// Interpret the parser table taking input from the given lexer, using the given lex buffer, and the given start state.
     /// Returns an object indicating the final synthesized value for the parse.
-    member Interpret :  lexer:(LexBuffer<char> -> 'tok) * lexbuf:LexBuffer<char> * startState:int -> obj 
+    member Interpret :  lexer:(LexBuffer<'char> -> 'tok) * lexbuf:LexBuffer<'char> * startState:int -> obj 
 
-/// Indicates an accept action has occurred
+#if INTERNALIZED_FSLEXYACC_RUNTIME
```

Comment:
`INTERNALIZED_FSLEXYACC_RUNTIME` in https://github.com/fsprojects/FsLexYacc mean the types are internal. 
Without types are public and used for `FsLexYacc.Runtime` assembly
fslex/fsyacc in https://github.com/fsprojects/FsLexYacc can use the `FsLexYacc.Runtime` assembly or expect source files
@dsyme said to use source files directly (like current visualfsharp) instead of add a dependency to `FsLexYacc.Runtime.dll` assembly